agent/pkg/k8s/k8s.go

```diff
@@ -73,6 +73,13 @@ const (
 	Pod         = "Pod"
 	ReplicaSet  = "ReplicaSet"
 	StatefulSet = "StatefulSet"
+
+	// AgentNamespace is the namespace where the Buoyant Cloud agent is installed.
+	AgentNamespace = "buoyant-cloud"
```

Comment:
the cli already relies on `agent/pkg/k8s`, so i think we could remove `cli/pkg/k8s.Namespace` and reference this one instead.